### ISSUE TITLE
cli parameter "--exclude-linter" not working on version 0.44.0

### ISSUE THREAD

**Issue by geoff-grav** [brigade/scss-lint#718]
The title says it all.  
I get no errors, it just doesn't exclude any linters that I place in the parameter. The yml file works, but the cli command does not.  I think this is the same for "--include-linter".
However, reverting back to version 0.38.0 works fine.

`gem uninstall scss_lint`
`gem install scss-lint -v 0.38.0`

I am using Mac OSX 10.11

**Comment by connorshea** [comment]
Just ran into this same issue with include-linter not working.

- OS X 10.11
- ruby 2.3.0p0
- scss-lint 0.45.0

**Comment by bensaufley** [comment]
Confirmed in v0.47.1
- OS X 10.11.3
- Ruby 2.3.0p0

**Comment by bernk** [comment]
Neither include nor exclude is working for me in 0.48.0. It would also be nice if the readme was more explicit on how to use this, because I'm not 100% sure I'm doing it right.

`scss-lint file.scss -x HexNotation`

Is that the expected use?

**Comment by ibrahima** [comment]
I agree @bernk . I think the above is right because if you change the name of the linter (eg. hex_notation) you'll get an error about the linter not existing. And I also can confirm that --include-linters does not seem to be working for me in v0.48.0

**Comment by lotap** [comment]
I'm also experiencing this issue with v0.49.0. I was trying to pass `--exclude-linter NameFormat` as an argument via the Atom integration. When that didn't work, I tried in the console and still got warnings from any linter I tried to exclude. Creating a config yml with the options configured did successfully exclude the linter, though.

OSX 10.11.4
Ruby 2.3.0p0
scss-lint 0.49.0

**Comment by ibrahima** [comment]
Found the issue, the problem is that a hash literal is being merged with the current linter options and returned, instead of returning the current linter options. See https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/config.rb#L273

There might be a fancier way of fixing the issue, but the simplest approach seems best to me.
